galgebra/mv.py

```diff
@@ -180,6 +180,12 @@ def characterise_Mv(self):
     # as the part of the name after `_make_` is public API via the string
     # argument passed to __init__.
 
+    def _make_blade(ga, *kargs, **kwargs):
```

Comment:
```suggestion
    @staticmethod
    def _make_blade(ga, *kargs, **kwargs):
```

Reply:
This will be enough to make it pass, but it would be great if you could apply the approach from #80 here too

Reply:
Which would probably be
```
@staticmethod
def _make_blade(ga, __name, __grade):
    if isinstance(__name, str) and isinstance(__grade, int):
        return reduce(Mv.__xor__, [ga.mv('%s%d' % (__name, i), 'vector') for i in range(__grade)], ga.mv(1, 'scalar')).obj
    else:
        raise TypeError("name must be a string and grade must be an int")
```

Reply:
Just being curious. Why are you using __ to prefix argument names ?

Reply:
To prevent them being passed by name, which seems to be what the intent of the original API was. There's a comment about it in this file, a few lines up.

Reply:
Interesting !